okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -954,19 +959,38 @@ private void proxyInternal(Iterator<ModuleInstance> it,
     );
   }
 
+  static Buffer compressInternalResponse(RoutingContext ctx, Buffer respBuf) {
```

Comment:
What advantages has this code over Vert.x'/Netty's [setCompressionSupported](https://vertx.io/docs/apidocs/io/vertx/core/http/HttpServerOptions.html#setCompressionSupported-boolean-)?
https://vertx.io/docs/vertx-core/java/#_http_compression

Reply:
Only that okapi does not try to compress responses from modules.. In this PR that choice is left to the module to decide. Just as Okapi doesn't do for own handled responses less than some size. I'll make a branch with server compresion as well (did that in #1213 but that *also* used http compression for Okapi client).. The worst for all parties is if okapi uncompresses responses from backend modules.. then compressed them again.. then let the client calling okapi decompress .. More CPU work for no gain.